R/geom_recessions.R

```diff
@@ -416,11 +439,11 @@ GeomRecessionsText <- ggproto(
 #' @return A tibble with the following variables: \itemize{ \item
 #'   \code{start_char, end_char}: Chr. Easily readable labels for the beginning
 #'   and end of the recession \item \code{start_num, end_num}: Double. Dates
-#'   expressed as years, with decimels referring to months. (e.g. April = 4/12 =
+#'   expressed as years, with decimals referring to months. (e.g. April = 4/12 =
 #'   .333) \item \code{start_date, end_date}: Date. Dates expressed in R
 #'   datetime format, using the first day of the specified month. }
 #'
-#' @source \url{https://www.nber.org/cycles/NBER chronology.xlsx}
+#' @source \url{https://www.nber.org/data/cycles 'cycle dates pasted.csv'}

```

Comment:
we might want to clarify our definition of the `url` param now that we're using a different file than the standard one a user would find by just browsing the NBER website.

Reply:
OK I've updated it. Although as an FYI, this is linked to on the business cycles page - it's the link to "JSON File Format" [here](https://www.nber.org/research/data/us-business-cycle-expansions-and-contractions).